blockcypher/src/endpoint/balance.ts

```diff
@@ -28,25 +30,45 @@ const getChainId = (coin: CoinType, chain: ChainType): string => {
   }
 }
 
-const getBalance: balance.GetBalance = async (account, config) => {
-  const chainId = getChainId(account.coin as CoinType, account.chain as ChainType)
-  const api = new bcypher(account.coin, chainId, config.apiKey)
+const getBalances: balance.GetBalances<Config> = async (accounts, config) => {
+  const addresses = accounts.map((a) => a.address)
+  const { coin, chain } = accounts[0]
+  const chainId = getChainId(coin as CoinType, chain as ChainType)
+  const api = new bcypher(coin, chainId, config.apiKey)
   const params = { confirmations: config.confirmations }
-  const _getAddrBal = (): Promise<AddressBalance> =>
-    new Promise((resolve, reject) => {
-      api.getAddrBal(account.address, params, (error: Error, body: AddressBalance) =>
-        error ? reject(error) : resolve(body),
+
+  const _getAddrBal = (addrs: string[]) =>
```

Comment:
Underscores mean unused in JS-land (by linters and tsc too), we shouldnt be using it as an internal function naming convention. Best to keep regular camel casing.